json_serializable/lib/src/json_part_builder.dart

```diff
@@ -20,6 +20,13 @@ import 'json_serializable_generator.dart';
 /// increase the code size, but it may improve runtime performance, especially
 /// for large object graphs.
 ///
+/// May set [requireLibraryDirective] to `false` in order to opt-in to
+/// supporting a `1.25.0` feature of `part of` being usable without an  explicit
```

Comment:
`1.25.0` was never publish was it? Do we want to refer to `2.0.0` instead?

Reply:
ack!

Reply:
Fixed. Also: https://github.com/dart-lang/source_gen/issues/288